modules/rubiconBidAdapter.js

```diff
@@ -22,6 +24,10 @@ const VIDEO_ENDPOINT = '//fastlane-adv.rubiconproject.com/v1/auction/video';
 
 const TIMEOUT_BUFFER = 500;
 
+var ACCOUNT_ID;
+getConfig('rubiconAdapter', config => ACCOUNT_ID = config.accountId);
```

Comment:
this looks..... well, very dangerous. By setting a variable in an async callback, you've inherently created a race condition. This works (I would guess?) because a user will (should?) call `setConfig` sometime between the time that this code runs, and when they kick off an auction... but if someone tried to use `ACCOUNT_ID` in this file outside of those auction-related functions, you could have some really tricky bugs to track down.

Part of me wants to say "your adapter, your business :p" but.... given that you're prototyping an architecture here, I'm a bit concerned because I can't really see a way to do it safely.

It seems to me like it would have to look something like:

```
getConfig('myNamespace', config => {
  var property = config.myParam;
  module.exports = stuffWhichUses(property);
});
```

...which I'm pretty sure modules don't actually let you do.

Can you write an example of a very small module which reads a value from the config, and then exports something useful once the param has been read? Without assuming that callers follow that implicit `import files, setConfig, callUsefulMethod` order.